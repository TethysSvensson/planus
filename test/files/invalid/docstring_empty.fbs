//! Okay1
/// Bad1
//! Bad2
