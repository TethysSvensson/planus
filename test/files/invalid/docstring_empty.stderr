error: Doc comments are not meaningful here.
  ┌─ test/files/invalid/docstring_empty.fbs:2:1
  │
2 │ /// Bad1
  │ ^^^^^^^^

error: Inner doc comments are not meaningful here.
  ┌─ test/files/invalid/docstring_empty.fbs:3:1
  │
3 │ //! Bad2
  │ ^^^^^^^^

Error: Bailing because of errors
